buildtools/gradle/gradle.go

```diff
@@ -145,7 +145,7 @@ func (s ShellCommand) DependencyTasks() ([]string, error) {
 	var projects []string
 	lines := strings.Split(stdout, "\n")
 	for _, line := range lines {
-		if i := strings.Index(line, ":dependencies"); i != -1 {
+		if i := strings.Index(line, ":dependencies -"); i != -1 {
```

Comment:
This feel like a bit of a hack, although I somewhat believe parsing `gradle tasks` is a hack in itself so I don't have a problem with this as long as its tested, which you did. You should be aware of the command `gradle projects` which might be more the  more applicable command to run here.

Reply:
I wonder why `gradle tasks` was preferred when this part was first implemented

Reply:
I think it is much easier to parse and assures we get projects with `:dependencies` tasks.